### DISCUSSION TITLE
linter errors for steps.d.ts default setup

### DISCUSSION THREAD

**Discussion by alaz-aura** [codeceptjs/CodeceptJS#4122] [General]
Wondering if the linter errors should still apply for the steps code that's generated during `npx codedeptjs init`, or I simply ignore this file from the linter? (which defeats the purpose). Any other suggestions?

**steps.d.ts**
```
/// <reference types='codeceptjs' />
type steps_file = typeof import('./steps_file');

declare namespace CodeceptJS {
  interface SupportObject {
    I: I;
    current: any;
  }
  interface Methods extends REST {} > error found here
  interface I extends ReturnType<steps_file> {} > error found here
  namespace Translation {
    interface Actions {} > error found here
  }
}
```

**linter error:**
   9:13  error  An interface declaring no members is equivalent to its supertype  @typescript-eslint/no-empty-interface
  10:13  error  An interface declaring no members is equivalent to its supertype  @typescript-eslint/no-empty-interface
  12:15  error  An empty interface is equivalent to `{}`                          @typescript-eslint/no-empty-interface

**Comment by kobenguyent** [comment]
I guess so, we could ignore this file.

**Reply by alaz-aura** [reply]
cool, that's what I've done in the meantime. My projects linter wasn't happy with the interface errors shared above
